test/helpers.go

```diff
@@ -51,3 +53,24 @@ func InitConfigForIntegrationsTesting() {
 	// Load normal config from environment variables and config files
 	cmdutil.InitConfig()
 }
+
+func MockEventContainerV2(key string) eventsapi.EventContainer {
+	eventContainer := eventsapi.EventContainer{
+		EventVersion: eventsapi.EventVersion2,
+	}
+
+	eventV2 := eventsapi.EventV2{
+		RoutingKey:  key,
+		EventAction: "trigger",
+		Payload: eventsapi.PayloadV2{
+			Summary:  "Test summary",
+			Source:   "Test source",
+			Severity: "Error",
+		},
+	}
+
+	jsonEvent, _ := json.Marshal(eventV2)
```

Comment:
I don't think you actually need to marshal / unmarshal here since `json.RawMessage` is actually a `byte[]`.

Reply:
Good catch! Forgot to change this when we switch to `json.RawMessage`: 2f72dfa